amundsen_application/static/js/components/SearchPage/SearchPanel/index.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { ResourceType } from 'interfaces/Resources';
+import { GlobalState } from 'ducks/rootReducer';
+import { DashboardSearchResults, TableSearchResults, UserSearchResults } from 'ducks/search/types';
+import { TABLE_RESOURCE_TITLE, USER_RESOURCE_TITLE } from 'components/SearchPage/constants';
+import AppConfig from 'config/config';
+
+import './styles.scss';
+
+export interface StateFromProps {
+  selectedTab: ResourceType,
+  tables: TableSearchResults;
+  dashboards: DashboardSearchResults;
+  users: UserSearchResults;
+}
+
+export interface OwnProps {
+  onTabChange: (tab: ResourceType) => void;
+}
+
+export type SearchPanelProps = StateFromProps & OwnProps;
+
+export class SearchPanel extends React.Component<SearchPanelProps> {
+
+  constructor(props) {
+    super(props);
+  }
+
+  onChange = (event) => {
+    this.props.onTabChange(event.target.value);
+  };
+
+  renderRadioOption = (value: ResourceType, label: string, count: number) => {
+    return (
+      <div className="radio">
+        <label className="radio-label">
+          <input
+            type="radio"
+            name="resource"
+            value={ value }
+            checked={ this.props.selectedTab === value }
+            onChange={ this.onChange }
+          />
+          <span className="subtitle-2">{ label }</span>
+          <span className="body-secondary-3 pull-right">{ count }</span>
+        </label>
+      </div>
+    );
+  };
+
+
+  render = () => {
+    return (
+      <div className="search-control-panel">
+        <div className="title-2">Resource</div>
```

Comment:
Is there a reason why this isn't it's own component?

Reply:
The logic seemed simple enough to include here, especially after implementing a RadioButtons component. I don't have strong feelings about this, will break out into separate component if you feel strongly about this.

Reply:
I think it will be easier to organize and more important test everything that will potentially go into the side panel if they are all broken up into different components. 

I see `SidePanel` as responsible for laying out everything inside of it and testing that they exist and subcomponents will test their own functionality and rendering. No need to change it in this PR though. Whoever add the filter component will likely want to make the change.

Reply:
Since it looks like we're going in that direction anyways, I'll break it out now.